engine/apps/schedules/tests/test_shift_swap_request.py

```diff
@@ -123,6 +123,9 @@ def test_take_own_ssr(shift_swap_request_setup) -> None:
         ssr.take(beneficiary)
 
 
+@pytest.mark.skip(
+    "Skipping as flaky based on time of day that the test runs. Example failure here https://github.com/grafana/oncall/actions/runs/5747168275/job/15577755519?pr=2725#step:5:1005"
+)
```

Comment:
@matiasb I had to temporarily skip this test. I couldn't quite figure out the time discrepancy. This seems similar to the `related_shifts` API test